src/plugins/ffe/ffe_access_database.py

```diff
@@ -39,14 +38,14 @@ def read_player_dicts(self) -> Iterator[dict[str, str | int | datetime | None]]:
             'ffe_licence': 'joueur.AffType',  # str
         }
         # the fields for players with a club
-        club_fields: dict[str, tuple[str, types.FunctionType]] = common_fields | {
+        club_fields: dict[str, str | int | datetime | None] = common_fields | {
```

Comment:
looks like it should be:
```suggestion
        club_fields: dict[str, str] = common_fields | {
```